madgiraffestudios/src/components/About/About.js

```diff
@@ -7,14 +7,14 @@ import './About.scss';
 const About = () => (
 
     <React.Fragment>
-        <div className="bg-light" id="topParent">
+        <div className="bg-light" id="aboutParent">
             <div className="container pb-2" id="about">
-                    <div className="text-center pt-5 mx-auto w-50">
+                    <div className="text-center pt-5 mx-auto">
                             <h1 className="font-weight-bold p-3 secondary-color">
                              Mad Giraffe STuDios
                             </h1>  
-                    <p className="font-weight-900 mb-5 studio-text">
-                    There’s no better way to start a conversation than with art. It allows tough conversations to be had, thus allowing us to grow . Art inspires me to make social change and inspire others to follow suit.
+                    <p className="mb-5 studio-text">
+                    There’s no better way to start a conversation than with art. It allows tough conversations to be had, thus allowing us to grow. <br/> Art inspires me to make social change and inspire others to follow suit.
```

Comment:
thanks for fixing this we can set this PR link with the issue I reassigned to you